scripts/stats-dashboard/mp-coverage.py

```diff
@@ -0,0 +1,263 @@
+#!/usr/bin/env python3
+"""
+Calculates MP coverage and generates plots
+Should be run by `./generate-markdown.py`
+"""
+from tqdm import tqdm
+import argparse
+import datetime as dt
+import os
+import pandas as pd
+
+
+
+
+here = os.path.dirname(__file__)
+
+ledamot_map = {
+    "fk": "förstakammarledamot",
+    "ak": "andrakammarledamot",
+    "ek": "ledamot"
+}
+
+skip = [
```

Comment:
Why skip these?

Reply:
They're like test protocols -- either we remove them or I can add this skip business to the protocol_iterators function in pyriksdagen

Reply:
Ok. I open up an issue on this then.